packages/app/cypress/e2e/specs_list_e2e.cy.ts

```diff
@@ -59,8 +61,8 @@ describe('App: Spec List (E2E)', () => {
   })
 
   it('shows a git status for each spec', () => {
-    cy.get('[data-cy="spec-list-file"]').each((row) => {
-      cy.wrap(row).contains('.git-info-row', 'Test Author')
+    cy.get('.git-info-row').each((row) => {
```

Comment:
Would be great to use a data selector instead of a class, but I realize this was already-existing